ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py

```diff
@@ -118,3 +118,55 @@ def test_tensorflow_shuffle(
         value=values[0],
         seed=seed,
     )
+
+
+# random_stateless_uniform
+@handle_frontend_test(
+    fn_tree="tensorflow.random.stateless_uniform",
+    shape=helpers.dtype_and_values(
+        available_dtypes=("int64", "int32"),
+        min_value=1,
+        max_value=5,
+        min_num_dims=1,
+        max_num_dims=1,
+        max_dim_size=9,
+    ),
+    seed=helpers.dtype_and_values(
+        available_dtypes=("int64", "int32"), min_value=0, max_value=10, shape=[2]
+    ),
+    minmaxval=helpers.get_bounds(dtype="int32"),
+    dtype=helpers.dtype_and_values(
+        available_dtypes=("int32", "int64", "float16", "float32", "float64"),
+        min_value=0,
+        max_value=0,
+        shape=[1],
+    ),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_stateless_uniform(
+    shape,
+    seed,
+    minmaxval,
+    dtype,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    dtype, _ = dtype
+    shape_input_dtypes, shape = shape
+    seed_input_dtypes, seed = seed
+
+    helpers.test_frontend_function(
+        input_dtypes=shape_input_dtypes + seed_input_dtypes,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        test_values=False,
+        shape=shape[0],
+        seed=(seed[0][0], seed[0][1]),
```

Comment:
I guess this is kind of redundant, why don't you use `seed[0]`? As per the [tensorflow docs](https://www.tensorflow.org/api_docs/python/tf/random/stateless_uniform) it is a `(2,)` tensor

Reply:
yes i agree, i have just fixed it